spec/models/money_group_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe MoneyGroup, type: :model do
```

Comment:
- [ ] Great adding unit tests for all the most important components of the RoR application. 👍🏻 Now, let's make sure to wad integration tests with capybara for all the most important components of the RoR application. ✍🏻 

![image](https://user-images.githubusercontent.com/79058364/221361585-894123ad-7e2c-4bb3-939a-7106066cc8c6.png)

Reply:
- [ ] just as previous reviewer mentioned, add unit and integration test for the important components of your application